test/dash/dash_parser_manifest_unit.js

```diff
@@ -1485,6 +1485,58 @@ describe('DashParser Manifest', () => {
     expect(minBufferTime).toBe(75);
   });
 
+  it('get manifest value if ignoreMaxSegmentDuration is false', async () => {
```

Comment:
This test and the next one are nearly identical. You could make a helper function that takes `ignoreMaxSegmentDuration` and `maxSegmentDuration` as parameters, and both save room and also make the relationship between the tests more explicit.